examples/ensure-name-substring/imperative/README.md

```diff
@@ -9,7 +9,7 @@ running [`ensure-name-substring`] function imperatively.
 
 Get the config example and try it out by running the following commands:
 
-```sh
+```shell
 $ kpt pkg get https://github.com/GoogleContainerTools/kpt-functions-catalog.git/examples/ensure-name-substring/imperative
```

Comment:
Per style guide: Need exactly one command per block, otherwise copy button doesn't work. It's good to provide a one-sentence description of each command anyways.

Reply:
> Need exactly one command per block, otherwise copy button doesn't work.
Good to know.

I guess we can migrate the changes in this PR to https://github.com/GoogleContainerTools/kpt-functions-catalog/pull/376, since it is split the getting package part out and move it to the front.